integration-tests/features/steps/stack_analyses_v2.py

```diff
@@ -79,6 +79,43 @@ def post_request(context, ecosystem, manifest, with_user_key, with_valid_user_ke
     context.response = response
 
 
+def post_request_with_transitives(context, ecosystem, manifest, with_user_key, with_valid_user_key):
```

Comment:
Transitive is just another parameter, can we add it to existing function instead of creating a new one.